src/content/en/2020/jamstack.md

```diff
@@ -3,21 +3,265 @@
 part_number: III
 chapter_number: 17
 title: Jamstack
-description: Jamstack chapter of the 2020 Web Almanac covering the use of JamStack across the web including static site generators, static site hosting and headless CMSss
+description: Jamstack chapter of the 2020 Web Almanac covering the use of Jamstack, the performance of popular Jamstack frameworks, as well as an analysis of real-user experience using the Core Web Vitals metadata.
 authors: [ahmadawais]
-reviewers: [tpiros, phacks, YuLingCheng]
 analysts: [denar90, remotesynth]
+reviewers: [maedahbatool, phacks]
 translators: []
-#ahmadawais_bio: TODO
+ahmadawais_bio: Ahmad Awais is an award-winning open-source engineer, Google Developers Expert Dev Advocate, Node.js Community Committee Outreach Lead, WordPress Core Dev, and VP of Engineering DevRel at WGA. He has authored various open-source software tools used by millions of developers worldwide. Like his <a href="https://shadesofpurple.pro/more">Shades of Purple</a> code-theme or projects like the <a href="https://github.com/AhmadAwais/corona-cli">corona-cli</a>. Awais loves to teach. Over 20,000 developers are learning from his <a href="https://AhmadAwais.com/courses">courses</a> i.e. <a href="https://nodecli.com/">Node CLI</a>, <a href="https://vscode.pro/">VSCode.pro</a>, and <a href="https://nextjsbeginner.com/">Next.js Beginner</a>. Awais received FOSS community leadership recognition as one of the <a href="https://ahmadawais.com/github-stars/">12 featured GitHub Stars</a>. He is a member of the SmashingMagazine Experts Panel; featured &amp; published author at CSS-Tricks, Tuts+, Scotch.io, SitePoint. You can mostly find him on Twitter <a href="https://twitter.com/MrAhmadAwais/">@MrAhmadAwais</a> where he tweets his <a href="https://awais.dev/odmt">#OneDevMinute</a> developer tips.
 discuss: 2053
 results: https://docs.google.com/spreadsheets/d/1BCC5Q4tePpTl8TiaGmSxBc9Lh2to7xBfVPMULFOBwvk/
 queries: 17_Jamstack
-#featured_quote: TODO
-#featured_stat_1: TODO
-#featured_stat_label_1: TODO
-#featured_stat_2: TODO
-#featured_stat_label_2: TODO
-#featured_stat_3: TODO
-#featured_stat_label_3: TODO
-unedited: true
+featured_quote: Stats suggest almost twice as many Jamstack sites exist now than in 2019. Developers enjoy a better development experience but that doesn't always translate to a better user experience. Let’s work towards continued progress for improving the real-user experience of browsing Jamstack sites.
+featured_stat_1: 154%
+featured_stat_label_1: more mobile websites built with Jamstack in 2020
+featured_stat_2: 1 gram
+featured_stat_label_2: CO2 is emitted by the median Jamstack page load
+featured_stat_3: 38%
+featured_stat_label_3: Jamstack sites are built using Next.js (three times more than any other framework!)
```

Comment:
This is what it looks like:

![Featured Stats](https://user-images.githubusercontent.com/10931297/101237541-8ad1c480-36d1-11eb-8250-b553d8c50c87.png)

Change to label sense, and shorten:

```suggestion
featured_stat_1: 154%
featured_stat_label_1: Increase in Jamstack websites in 2020
featured_stat_2: 1 gram
featured_stat_label_2: CO2 emitted the median Jamstack on page load
featured_stat_3: 38%
featured_stat_label_3: Jamstack sites built using Next.js
```

This would look like this:

![Feratured Stats (editted)](https://user-images.githubusercontent.com/10931297/101237643-6fb38480-36d2-11eb-9bfc-a6d11c893ca4.png)